api/getProfile/index.ts

```diff
@@ -1,19 +1,15 @@
 import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import { UserInfo } from "../models/UserInfo";
 
 const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
-    context.log('HTTP trigger function processed a request.');
-    const name = req.params.userId;
+    const userId = req.params.userId;
+    const userInfo: UserInfo = <UserInfo>(context.bindings.userInfo);

```

Comment:
```suggestion
    const userInfo = context.bindings.userInfo as UserInfo
```
Very interesting type casting syntax